Utilities/aliceHLTwrapper/test/testMessageFormat.cxx

```diff
@@ -17,7 +17,7 @@
 #include "Headers/DataHeader.h"
 #include "Headers/HeartbeatFrame.h"
 
-namespace o2::AliceHLT {
+namespace o2 { namespace AliceHLT {
```

Comment:
I think there is nothing in the conventions which applies directly to this, maybe the "One statement per loine"-recommendation, but I would also open each namespace on a separate line.